tests/system/action/mediafile/test_delete.py

```diff
@@ -0,0 +1,36 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MediafileDeleteActionTest(BaseActionTestCase):
```

Comment:
Please add one test with children to check that they are deleted, too.

Reply:
Rebased and Added two tests with children.

Reply:
Done.